installer/resources/lambda_rule_engine/files/rule_engine_cloudwatch_azure_rules.json

```diff
@@ -2621,6 +2621,27 @@
       "modifiedDate": "2022-08-23",
       "severity": "medium",
       "category": "security"
+    },
+    {
+    "ruleId": "TLS_Version_MySQL_Flexible_Server",
+    "ruleUUID": "TLS_Version_MySQL_Flexible_Server",
+    "policyId": "Enable_latest_TLS_version",
+    "ruleName": "minTLSVersion1.2",
+    "targetType": "mysqlflexible",
+    "assetGroup": "Azure",
+    "alexaKeyword": "MysqlFlexibleTLS",
+    "ruleParams": "{\"params\":[{\"encrypt\":false,\"value\":\"check-for-tls-version-mysql-flexible-server\",\"key\":\"ruleKey\"},{\"encrypt\":false,\"value\":\",\",\"key\":\"splitterChar\"},{\"encrypt\":false,\"value\":\"Application,Environment,Stack,Role\",\"key\":\"mandatoryTags\"},{\"encrypt\":false,\"value\":\"medium\",\"key\":\"severity\"},{\"encrypt\":false,\"value\":\"security\",\"key\":\"ruleCategory\"},{\"encrypt\":false,\"value\":\"\",\"key\":\"ruleOwner\"}],\"environmentVariables\":[],\"ruleId\":\"TLS_Version_MySQL_Flexible_Server\",\"autofix\":false,\"alexaKeyword\":\"MysqlFlexibleTLS\",\"ruleRestUrl\":\"\",\"targetType\":\"mysqlflexible\",\"pac_ds\":\"azure\",\"policyId\":\"Enable_latest_TLS_version\",\"assetGroup\":\"azure\",\"ruleUUID\":\"TLS_Version_MySQL_Flexible_Server\",\"ruleType\":\"ManageRule\"}",
+    "ruleFrequency": "0 0/6 * * ? *",
+    "ruleExecutable": "",
+    "ruleRestUrl": "",
+    "ruleType": "ManageRule",
+    "ruleArn": "arn:aws:events:us-east-1:***REMOVED***:rule/TLS_Version_MySQL_Flexible_Server",
+    "status": "ENABLED",
+    "userId": "ASGC",
+    "displayName": "TLS version for MySQL Flexible Database Server should be 1.2",
```

Comment:
Lets modify displayname to use 'latest tls version' instead of 1.2

Reply:
changed name